workflows/tree-from-protein.cwl

```diff
@@ -7,10 +7,15 @@ requirements:
   - class: SubworkflowFeatureRequirement
 
 inputs:
+  label: UniProt identifier
+  doc: Enter a UniProt identifier
+  default: uniprot:wap_rat
```

Comment:
This will not work, see https://github.com/common-workflow-language/cwltool/issues/530